### ISSUE TITLE
Add an ability to reset zoom

### ISSUE THREAD

**Issue by eyalroth** [microsoft/terminal#2578]
# Description of the new feature/enhancement

Allow to reset the zoom (to 100%) after changing it with `Ctrl + Mouse scroll`. A keyboard shortcut -- say, `Ctrl+0` -- would be enough to start with.

There is already [an issue](https://github.com/microsoft/terminal/issues/300) for this request, but it mainly addresses another bug.

**Comment by Jaykul** [comment]
Just a note: for now, you can work around this by triggering a settings reload. E.g. by `touch`ing the file...

```PowerShell
sp "~\AppData\Local\Packages\Microsoft.WindowsTerminal_*\LocalState\profiles.json" LastWriteTime (Get-Date)
```

**Comment by zoechi** [comment]
Ctrl+0 works for me.
It's weird though that ctrl+- zooms out, ctrl++ doesn't allow to zoom in.

I'm searching for a way to disable zooming at all.

**Comment by zadjii-msft** [comment]
@zoechi for the record, the reason "increaseFontSize" is bound to <kbd>Ctrl+=</kbd> is that on a US keyboard layout, `+` is on the same key as `=`, but you need to hold shift to get the `+`. So typically, most applications has zoom in bound to <kbd>Ctrl+=</kbd> anyways, since it's silly to have zoom out bound to `ctrl+key` and zoom in bound to `ctrl+shift+key`

<hr>

Goodness gracious, this was actually added in #3505, which was shipped in 0.7. Looks like we let #3319 exist without getting duped to this thread.

**Comment by zoechi** [comment]
@zadjii-msft Thanks for the info. I wasn't aware of that because on German keyboards it's usually <kbd>Ctrl</kbd>+<kbd>+</kbd>
